README.md

```diff
@@ -1,2 +1,42 @@
-# bandit-action
-GitHub Action for Bandit
+# Bandit by PyCQA
+
+The official [Bandit](https://github.com/PyCQA/bandit) Action developed by the [PyCQA](https://github.com/PyCQA).
+
+## Usage
+
+Here is a minimal complete example workflow to create a Code Scanning action using Bandit.
+
+```yaml
+name: Bandit
+
+on:
+  workflow_dispatch:
+
+jobs:
+  analyze:
+    runs-on: ubuntu-latest
+    permissions:
+      # required for all workflows
+      security-events: write
+      # only required for workflows in private repositories
+      actions: read
+      contents: read
+    steps:
+      - name: Perform Bandit Analysis
+        uses: PyCQA/bandit-action@v1
+```
+
+## Inputs
+
+| Name | Description | Required | Default Value |
```

Comment:
Man, I wish Markdown wasn't so ugly to read as plain-text like this is. It makes it impossible to review

Reply:
Yeah, RST can be a little cleaner with its list table format.